pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -40,9 +40,30 @@ const (
 var ErrFileNotFound = errors.New("file not found")
 
 func stringifyAddress(addr *objectSDK.Address) string {
-	h := sha256.Sum256([]byte(addr.String()))
+	return addr.ObjectID().String() + "." + addr.ContainerID().String()
+}
+
+func addressFromString(s string) (*objectSDK.Address, error) {
```

Comment:
May it be simpler to reverse the address string? It will be less code to understand, and maybe it could work faster a little.